.bazelrc

```diff
@@ -109,3 +109,11 @@ test --define=MEDIAPIPE_DISABLE=1
 build --cxxopt=-DMEDIAPIPE_DISABLE=1
 coverage --cxxopt=-DMEDIAPIPE_DISABLE=1
 test --cxxopt=-DMEDIAPIPE_DISABLE=1
+
+build --define=PYTHON_DISABLE=1
+coverage --define=PYTHON_DISABLE=1
+test --define=PYTHON_DISABLE=1
+
```

Comment:
We could add flag MEDIAPIPE_DISABLE here as well?

Eventually we should switch this flag name to OVMS_MEDIAPIPE_DISABLE to be able to upstream at some point.

Reply:
flag is added before this line.